### DISCUSSION TITLE
Lookup inference

### DISCUSSION THREAD

**Discussion by cbizon** [NCATSTranslator/TranslatorArchitecture#66] [General]
When responding to a lookup query, should the client have the ability to turn off descendent inferences? What about Biolink model inference vs. ontology subclass inference? Control separately? Only control subclass inference?

**Comment by remontoire-pac** [comment]
Assuming we are also talking about conflation inference in addition to
those two.
Client=ARA to KP, yes, separate controls.
Client=end-user, maybe not control over all 3.
